README.md

```diff
@@ -78,6 +78,23 @@ The `cosi` application provides both a client for signing messages,
 and a server implementing the cosigner or witness-server role
 in the CoSi protocol.
 
+## Chose the right directory for the examples
+
+For the examples in the following sections, we suppose you're in a directory
+where you can find the following files: `README.md` and `dedis-group.toml`.
+
+If you compiled from source, please change the directory like so:
+
+```bash
+cd $GOPATH/src/github.com/dedis/cosi
+```
+
+If you used the binary distribution, please use
+
+```bash
+cd $( dirname $( which cosi ) )
```

Comment:
Not sure. Everything else as described below also won't work in this case (calling `cosi` won't work if it's not in the path)